content/_releases/2020-08-27.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Product Update - v20.35
+description: Details of the product update v20.35 from 27th of August 2020.
+layout: rel
+version: "20.35"
+section: Product Updates
+category: product-updates
+releaseDate: 2020-08-27
+---
+
+
+## Features
+
+### Recipes
+
+With this release we have revitalised the Recipes functionality of our platform.
+It underwent changes under the hood ([the Recipes API](https://api-sparrow.elastic.io/docs/v2/#recipes-(experimental))) as well as in the UI. For now the feature is in rapid advancement/development phase.
```

Comment:
`(https://api-sparrow.elastic.io/docs/v2/#recipes-(experimental))`  -- remove api-sparrow and use base_url

Reply:
upsi, thanks for noticing